packages/app/features/dashboard/profile/textEllipsis.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+import { insert, join, pipe, remove, split } from "ramda";
+import { Text, View } from "app/design-system";
+import { tw } from "app/design-system/tailwind";
+
+export function TextEllipsis({ text }: { text: string }) {
+  const [wrapperWidth, setWrapperWidth] = useState(0);
+  const [displayedText, setDisplayedText] = useState("A");
```

Comment:
```suggestion
  const [displayedText, setDisplayedText] = useState(text);
```

Reply:
The text can wrap messing everything up, if we tell it not to wrap it will expand its parent messing everything again.

Reply:
well initializing with `A` looks a little bit messy in my opinion. What about an empty string?

Reply:
We need to somehow calculate length of one character so it can't be an empty string. I can extract it to some const like `initialDisplayedText` or take some character from text `text.substring(0, 4)`

Reply:
hmm, it looks like a bit hacky solution to me anyway. What if we set in parent overflow to `hidden`?